package.json

```diff
@@ -47,7 +50,8 @@
     "start": "react-scripts start",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "build": "cross-env GENERATE_SOURCEMAP=false react-scripts build"
+    "build": "cross-env react-scripts build || npm run sentry:sourcemaps",
+    "sentry:sourcemaps": "./node_modules/.bin/sentry-cli sourcemaps inject --org it-university-of-copenhagen --project zeeguu-web ./build && ./node_modules/.bin/sentry-cli sourcemaps upload --org it-university-of-copenhagen --project zeeguu-web ./build"
```

Comment:
i'm a bit puzzled as to why do I see these changes coming from you. I thought I have made them?